tests/unit/s2n_psk_test.c

```diff
@@ -999,5 +999,33 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(conn));
     }
 
+    /* Test s2n_psk_validate_keying_material */
+    {
+        /* Safety */
+        EXPECT_ERROR_WITH_ERRNO(s2n_psk_validate_keying_material(NULL), S2N_ERR_NULL);
+
+        struct s2n_connection *conn = s2n_connection_new(S2N_SERVER);
+        DEFER_CLEANUP(struct s2n_psk *chosen_psk = s2n_test_psk_new(conn), s2n_psk_free);
+
+        /* No-op if no chosen PSK */
+        EXPECT_OK(s2n_psk_validate_keying_material(conn));
+        conn->psk_params.chosen_psk = chosen_psk;
```

Comment:
Should this line come after the next comment?

(Ditto for all other lines after the `EXPECT` macros?